bait-files/src/main/webapp/submission.jsp

```diff
@@ -35,22 +35,105 @@
+                    <option value="North Carolina">North Carolina</option>
+                    <option value="North Dakota">North Dakota</option>
+                    <option value="Oklahoma">Oklahoma</option>
+                    <option value="Ohio">Ohio</option>
+                    <option value="Oregon">Oregon</option>
+                    <option value="Pennsylvania">Pennsylvania</option>
+                    <option value="Rhode Island">Rhode Island</option>
+                    <option value="South Carolina">South Carolina</option>
+                    <option value="South Dakota">South Dakota</option>
+                    <option value="Tennessee">Tennessee</option>
+                    <option value="Texas">Texas</option>
+                    <option value="Utah">Utah</option>
+                    <option value="Vermont">Vermont</option>
+                    <option value="Virginia">Virginia</option>
+                    <option value="Washington">Washington</option>
+                    <option value="West Virginia">West Virginia</option>
+                    <option value="Wisconsin">Wisconsin</option>
+                    <option value="Wyoming">Wyoming</option>
+                    <option value="Alberta">Alberta</option>
+                    <option value="British Columbia">British Columbia</option>
+                    <option value="Manitoba">Manitoba</option>
+                    <option value="New Brunswick">New Brunswick</option>
+                    <option value="Newfoundland and Labrador">Newfoundland and Labrador</option>
+                    <option value="Northwest Territories">Northwest Territories</option>
+                    <option value="Nova Scotia">Nova Scotia</option>
+                    <option value="Nunavut">Nunavut</option>
+                    <option value="Ontario">Ontario</option>
+                    <option value="Prince Edward Island">Prince Edward Island</option>
+                    <option value="Quebec">Quebec</option>
+                    <option value="Saskatchewan">Saskatchewan</option>
+                    <option value="Yukon">Yukon</option>
+                  </select>
                 </div>
-                <div id="map">
+                <div>
+                  <label for="postal-zip-code-input">Postal/Zip Code: </label>
+                  <input type="text" id="postal-zip-code-input" name="postal-zip-code-input">
                 </div>
-                <p>* Check address to determine if the address fields have been typed in correctly * </p>
-                <button type="button" onclick="placeMarkerOnUserInputtedAddress()">Check Address Location</button>
+              </div>
+              <div id="map">
+              </div>
+              <p>* Check address to determine if the address fields have been typed in correctly *</p>
+              <button type="button" onclick="placeMarkerOnUserInputtedAddress()">Check Address Location</button>
             </li>
```

Comment:
I think that hard coding is the best option here given the amount of time we have left!